ormar/models/mixins/save_mixin.py

```diff
@@ -402,3 +418,11 @@ def _get_field_values(self, name: str) -> List:
         if not isinstance(values, list):
             values = [values]
         return values
+
+    @classmethod
+    def get_fields_has_onupdate(cls) -> List[str]:
```

Comment:
Would rename this to `get_fields_with_onupdate`.

Also, when it's called the first time model should be fully populated (relations etc.) so we can cache those fields in a property `_on_update_fields` cause it will be run with each `update`/`bulk_update` and return from _cache if this attribute is not none to skip iterating through all fields every time. Something similar to `cls.extract_related_names()`